Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla43941.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
```

Comment:
Both this and the test case for 32206 could be automated. Take a look at the test for [44166](https://github.com/xamarin/Xamarin.Forms/blob/122f0e3e6060711d6c3b2b8523236b8a057434df/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla44166.cs) for an example.